mlir/test/Dialect/Linalg/linalg-morph-ops.mlir

```diff
@@ -0,0 +1,25 @@
+// Forward path `named -> category -> generic`
+// RUN: mlir-opt %s -linalg-morph-ops=named-to-category | FileCheck %s  --check-prefix=NAMED_TO_CATEGORY
+// RUN: mlir-opt %s -linalg-morph-ops=named-to-generic |  FileCheck %s  --check-prefix=NAMED_TO_GENERIC
+// RUN: mlir-opt %s -linalg-morph-ops=named-to-category |  \
+// RUN:   mlir-opt %s -linalg-morph-ops=category-to-generic | FileCheck %s  --check-prefix=CATEGORY_TO_GENERIC
```

Comment:
I would have this `prefix=NAMED_TO_GENERIC` to test the multi-step case, but create a new test for CATEGORY_TO_GENERIC.